src/utils.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const generatePayloads = require('./payloads');
+
+const generateScript = (host, port) => {
+	const payloads = generatePayloads(host, port);
+
+	return Object.keys(payloads).reduce((acc, cmd) => {
+		acc += `if type ${cmd} &> /dev/null; then\n` +
```

Comment:
I think maybe `command -v` would be a better option over `type`, thoughts?

https://unix.stackexchange.com/a/85250

Reply:
I looked into it and yes only for verification `command -v` is better.